rundeckapp/grails-spa/packages/ui-trellis/src/app/components/job/options/OptionEdit.vue

```diff
@@ -0,0 +1,1402 @@
+                    <label class="control-label">{{
+                      $t(
+                        "form.option.valuesType.url.authentication.token.label",
+                      )
+                    }}</label>
+                  </div>
+                  <div class="col-md-8 input-group">
+                    <span
+                      class="input-group-addon has_tooltip"
+                      :title="$t('form.option.defaultStoragePath.description')"
+                    >
+                      <i class="glyphicon glyphicon-lock"></i>
+                    </span>
+
+                    <input
+                      type="text"
+                      class="form-control"
+                      name="remoteUrlBearerToken"
+                      v-model="option.configRemoteUrl.tokenStoragePath"
+                      size="20"
+                      placeholder=""
+                    />
+
+                    <span class="input-group-btn">
+                      <key-storage-selector
+                        v-model="option.configRemoteUrl.tokenStoragePath"
+                        :storage-filter="'Rundeck-data-type=password'"
+                        :allow-upload="true"
+                        :read-only="false"
+                      />
+                    </span>
+                  </div>
+                </div>
+                <!--bearerToken AUTH-->
+              </div>
+            </div>
+          </div>
+          <div v-else-if="option.valuesType && optionValuesPlugins">
+            <plugin-info
+              :detail="getProviderFor(option.valuesType)"
+              :show-icon="false"
+              :show-title="false"
+              :show-description="true"
+              :show-extended="true"
+              description-css="help-block"
+            >
+            </plugin-info>
+          </div>
+        </div>
+      </div>
+      <div class="form-group">
+        <label class="col-sm-2 control-label">{{
+          $t("form.option.sort.label")
+        }}</label>
+
+        <div class="col-sm-3">
+          <div class="radio radio-inline">
+            <input
+              type="radio"
+              name="sortValues"
+              :value="false"
+              v-model="option.sortValues"
+            />
+            <label for="option-sort-values-no">
+              {{ $t("no") }}
+            </label>
+          </div>
+          <div class="radio radio-inline">
+            <input
+              type="radio"
+              name="sortValues"
+              :value="true"
+              v-model="option.sortValues"
+            />
+            <label for="option-sort-values-yes">
+              {{ $t("yes") }}
+            </label>
+          </div>
+          <div class="help-block">
+            {{ $t("form.option.sort.description") }}
+          </div>
+        </div>
+
+        <div
+          class="input-group col-sm-3"
+          :class="{ 'has-error': hasError('delimiter') }"
```

Comment:
```suggestion
          :class="{ 'has-error': hasError('multivaluedDelimiter') }"
```

Note: need to change this also from the BE, as otherwise, both inputs for delimiters get marked as with errors (API is returning the key delimiter in the messages, even though the error should be only in the multivalued one
 
![Screenshot 2024-02-26 at 4 21 23 PM](https://github.com/rundeck/rundeck/assets/139791797/a75b32e8-f454-4059-9caf-76e80e5177ab)

Reply:
thanks, i've corrected this, actually the other delimiter used the wrong field name check.  The results from the BE will match the input field names